server/command_create.go

```diff
@@ -83,6 +83,15 @@ func parseCreateArgs(args []string, install *Installation) error {
 		return err
 	}
 
+	// the filestore has a different default depending upon the target installation type
+	if install.Filestore == "" {
+		if install.License == "e20" {
```

Comment:
```suggestion
		if install.License == licenseOptionE20 {
```

Reply:
Ah, if that constant exists there's a few places that should be updated to use it actually, thanks